service/src/test/java/org/kiwiproject/champagne/resource/AuditRecordResourceTest.java

```diff
@@ -32,23 +34,29 @@
 @DisplayName("AuditRecordResource")
 @ExtendWith(DropwizardExtensionsSupport.class)
 class AuditRecordResourceTest {
-    
+
     private static final AuditRecordDao AUDIT_RECORD_DAO = mock(AuditRecordDao.class);
     private static final AuditRecordResource RESOURCE = new AuditRecordResource(AUDIT_RECORD_DAO);
 
     private static final ResourceExtension RESOURCES = ResourceExtension.builder()
-        .bootstrapLogging(false)
-        .addResource(RESOURCE)
-        .addProvider(JerseyViolationExceptionMapper.class)
-        .addProvider(JaxrsExceptionMapper.class)
-        .build();
+            .bootstrapLogging(false)
+            .addResource(RESOURCE)
+            .addProvider(JerseyViolationExceptionMapper.class)
+            .addProvider(JaxrsExceptionMapper.class)
+            .build();
 
     @RegisterExtension
     private final JwtExtension jwtExtension = new JwtExtension("bob");
 
+    @BeforeEach
+    void setUp() {
+        DeployableSystemThreadLocal.setCurrentDeployableSystem(1L);
```

Comment:
If this is used in many tests, we should create a Jupiter extension. This can be done as a follow on commit.